lib/rules/rules.json

```diff
@@ -214,6 +214,11 @@
         "alternative": "Object.entries()",
         "ES6": true
     },
+    "get": {
+        "compatible": false,
+        "alternative": "optional chaining to get nested values and nullish coalescing operator for fallback values",
+        "ES11": true
```

Comment:
I'm thinking this and `compatible` should be replaced by something like `browserslist`? Thoughts?

Reply:
Sounds good but I'm not that familiar with those configs. Maybe we should open another issue to discuss it more deeply and also change it in the other rules? What do you think?

Reply:
I agree